.github/workflows/README.md

```diff
@@ -12,19 +12,19 @@ and well-documented pattern colloquially called "Docker in Docker" or simply
 
 Building a Windows-based OCI image inside a Windows-based OCI container is
 _not_, as of this writing, a common, well-understood, or well-documented
-pattern. Building Brigade's one Windows-based OCI image, therefore, requires the
-use of a physical Windows machine or virtual machine having an appropriate
+pattern. Building Brigade's two Windows-based OCI images, therefore, requires
```

Comment:
Wonder if we want to just simplify to 'Building Brigade's Windows-based OCI images' (here and elsewhere)... unless we're pretty certain it'll now be capped at the two?

Reply:
Pretty sure it's limited to these two, but I like your suggestion regardless. Changing.

Reply:
All set.